src/converter/renderers/render-aem.js

```diff
@@ -78,7 +183,7 @@ async function transformAemPageMetadata(htmlString, params) {
 /**
  * @param {string} htmlString
  */
-function transformHTML(htmlString, aemAuthorUrl, path) {
```

Comment:
Was there a reason for the var name change?

Reply:
We have introduced path module to work with file system paths. So, renamed existing path var to pagePath to avoid conflicts. 
import path from 'path';

Reply:
Reverted the var name change